### DISCUSSION TITLE
Is there a strict limit on the number of team members (5)?

### DISCUSSION THREAD

**Discussion by shun-zheng** [PaddlePaddle/PaddleSpatial#7] [BaiduKDDCup2022] [answered]
Dear organizers,

Thanks for providing such a valuable dataset for contest.
I wonder whether there is a strict limit on the number of team members? We have more than 5 members who want to constitute a team.

Thanks

**Comment by zhoujingbo** [comment, marked as answer]
> Dear organizers,
> 
> Thanks for providing such a valuable dataset for contest. I wonder whether there is a strict limit on the number of team members? We have more than 5 members who want to constitute a team.
> 
> Thanks

Dear Sir/Mdm;

Thanks a lot for supporting the Baidu KDD Cup 2022. This year we set a strict limit for each time with a maximum of 5 members.  The Amazon KDD Cup 2022 also has the same setting.  Limiting the size of teams can make the competition result more comparable. Thanks a lot for your understanding.

Your friends can form another team to participate in the challenge. But note that "Each participant is only permitted to be in ONE team. Any participant that is part of more than one Team may be disqualified and her/his corresponding Team may be disqualified, if the challenge organizers identify and/or are reported with evidence for such cases."

Thanks a lot for your understanding and support.

Regards
Jingbo
